i18n/ja/docusaurus-plugin-content-docs/current/unraid-os/release-notes/6.12.0.md

```
@@ -1,6 +1,6 @@
 # バージョン 6.12.0 2023-06-14
 
-## アップグレードに関する注意事項 {/* #upgrade-notes */}
+## アップグレードに関する注意事項 {/* #upgrade-notes */} {/* #upgrade-notes */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)